pygfx/renderers/wgpu/meshshader.py

```diff
@@ -68,9 +68,18 @@ def get_bindings(self, wobject, shared):
         if getattr(geometry, "normals", None) is not None:
             normal_buffer = geometry.normals
         else:
-            normal_data = normals_from_vertices(
-                geometry.positions.data, geometry.indices.data
-            )
+            if geometry.indices.data.shape[1] == 4:
```

Comment:
```suggestion
            if geometry.indices.data.shape[-2] == 4:
```

Reply:
Do you mean -1?

Reply:
hmm I think it wont work for ndim >2 or? but anyways yes it should be -1

Reply:
You're right! I updated the suggestions